chandra_aca/tests/test_all.py

```diff
@@ -94,6 +94,21 @@ def test_mag_for_p_acq():
     assert np.allclose(mag, 10.821, rtol=0, atol=0.01)
 
 
+def test_halfwidth_adjustment():
+    for mag, mults in ((6.0, [1.0, 1.0, 1.0, 1.0]),
+                       (8.5, [1.0, 1.0, 1.0, 1.0]),
+                       (9.25, [0.9, 0.8, 0.7, 0.505]),
+                       (10.0, [0.8, 0.6, 0.4, 0.01]),
+                       (11.0, [0.01, 0.01, 0.01, 0.01])):
+        pacq = acq_success_prob(mag=mag, halfwidth=120)
```

Comment:
Yep, I misspoke in constructing the "worst" case, which would probably be for a 125 arcsec hw box.  I agree that this should be conservative and fine and was just double-checking it is what you had in mind.